app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def yes_no(boolean)
+    "<span class='#{ boolean ? "fa fa-check-circle has-text-success" : "fa fa-ban" }'></span>".html_safe
+  end
```

Comment:
This is a utility I find very useful, but not sure if it's an antipattern or something y'all would rather not use? (And, if ok to use, ok to include in this PR where I use it, or should these always be extracted?)